dev-lang/vala/vala-0.36.11.recipe

```diff
@@ -64,6 +73,20 @@ BUILD_PREREQUIRES="
 	cmd:pkg_config$secondaryArchSuffix
 	"
 
+defineDebugInfoPackage vala$secondaryArchSuffix \
+	$libDir/libvala-${portVersion%.*}.so.0.0.0 \
+	$binDir/vala \
+	$binDir/valac \
+	$binDir/vapicheck \
+	$binDir/vapigen \
+	$binDir/vala-${portVersion%.*} \
+	$binDir/valac-${portVersion%.*} \
+	$binDir/vapicheck-${portVersion%.*} \
+	$binDir/vapigen-${portVersion%.*}
+	# looks like it doesn't take more then one "-" here
+	#$binDir/vala-gen-introspect \
+	#$binDir/vala-gen-introspect-${portVersion%.*} \
```

Comment:
```diff
-	# looks like it doesn't take more then one "-" here
-	#$binDir/vala-gen-introspect \
-	#$binDir/vala-gen-introspect-${portVersion%.*} \
```
`vala-gen-introspect` is a symlink to `vala-gen-introspect-0.36`, which is itself just a shell script.
(So this is why it can't be included in the debuginfo package.)